doc/api/v0.yaml

```diff
@@ -441,3 +464,46 @@ components:
               type: array
               items:
                 $ref: "#/components/schemas/Proposal"
+
+    ReviewTag:
+      type: string
+      enum:
+        - Alignment
+        - Verifiability
+        - Feasibility
+        - Impact
+        - Auditability
+
+    AdvisorReview:
+      properties:
+        id:
+          type: integer
+          format: i32
+        proposal_id:
+          type: integer
+          format: i32
+        rating_given:
+          type: integer
+          format: i32
```

Comment:
It would be good to have a description field detailing the scale of the integer rating.
Also minimum and maximum values could be specced?

Reply:
yep, added now. Thanks :)